components/Footer/Footer.tsx

```diff
@@ -18,13 +18,19 @@ export const Footer = () => {
 							href="https://github.com/AccessibleForAll/AccessibleWebDev/blob/main/CONTRIBUTING.md"
 							className={styles.footerLink}>
 							Become a Contributor
+							<span>
+								<BsBoxArrowUpRight aria-label="Become a Contributor" />
+							</span>
```

Comment:
@EmmaDawsonDev kindly let me know if i can improve the aria-label description

Reply:
Yes, it should be “opens in new window” on both icons as that’s the function of the icon and preferably as an alt text instead of an aria label.

Reply:
Oh, interesting. Aria-label should be fine, I’ll just double check with a screen reader.